src/test/java/com/suse/pase/MainTest.java

```diff
@@ -35,7 +35,7 @@ static void setup() throws Exception {
     @org.junit.jupiter.api.Test
     public void searchTest() throws Exception {
         var patchPath = resourcePath.resolve("patches").resolve("CVE-2017-5638.patch");
-        var results = Search.search(indexPath, patchPath, false, false);
+        var results = Search.search(indexPath, patchPath, false, false, false);
```

Comment:
Ideally we should add tests for the new query types - better but not necessarily in this PR, just eventually.